### PR TITLE
abseil 20230125.0

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
`abseil` has c++ version support change.
```
  /opt/homebrew/include/absl/base/policy_checks.h:79:2: error: "C++ versions less than C++14 are not supported."
  #error "C++ versions less than C++14 are not supported."
```

File: Formula/libphonenumber.rb [hunk 1]
```diff
@@ -32,6 +32,8 @@ class Libphonenumber < Formula
 
   fails_with gcc: "5" # For abseil and C++17
 
+  patch :DATA
```
Comment (chenrui333) [inline Formula/libphonenumber.rb:35]: The CMAKE_CXX_STANDARD override somehow did not get applied.

**Comment by chenrui333** [comment]
Should be in a better spot now.

**Comment by chenrui333** [comment]
some networking failures

**Comment by chenrui333** [comment]
- mavsdk has some tag issue https://github.com/mavlink/MAVSDK/issues/1978

**Comment by chenrui333** [comment]
Should be a rerun after #122290

**Comment by carlocab** [comment]
Cherry-picked onto #123507.

**Pull request closed without merging**